src/utils/lagrange_poly.rs

```diff
@@ -0,0 +1,46 @@
+use ark_ff::PrimeField;
+use ark_poly::{univariate::DensePolynomial, DenseUVPolynomial};
+
+pub fn compute_lagrange_poly<F: PrimeField>(p_i: &[F]) -> DensePolynomial<F> {
```

Comment:
How would you see to add a comment here mentioning that this returns the Lagrange *interpolated* polynomial?
(as a sidenote: I'm thinking this to avoid confusion with the name of the method thinking that this returns the Lagrange polynomial basis, where it actually returns the interpolated polynomial through the combination of the Lagrange polynomial basis with the given evaluations)

Reply:
Yup this is relevant, added [here](https://github.com/privacy-scaling-explorations/folding-schemes/pull/47/commits/4d5fe2d3db817174e061e410eaf3e5d6d89ef01a). Also changed the name to be extra clear